2021/28xxx/CVE-2021-28715.json

```diff
@@ -1,18 +1,106 @@
 {
```

Comment:
{'version_affected': '?', 'version_value': 'consult Xen advisory XSA-392'} is not of type 'array'